### DISCUSSION TITLE
Addind different templates

### DISCUSSION THREAD

**Discussion by piettes** [tailpress/tailpress#158] [Q&A]
Hello,

I am trying to have different templates for my Wordpress theme, for example "homepage-template" & "about-template" 

But right now, if I add a template in `templates/homepage.html` and in the `theme.json`, and if I try to use it, I get a completely blank content, without any styling. It's as if `index.php`, `header.php` & `footer.php` are completely igrnored.

I can see in  `index.php` that the content `template-parts/content.php` is always used. Is there a way to change that dynamically ? 

What is the proper way to add template ? Is it supported with Tailpress ? 

Thank you very much !

**Comment by charlie-george-1989** [comment]
Link to repo please!
